src/components/InputTime/Dropdown/Dropdown.module.css

```diff
@@ -6,11 +6,15 @@
   margin-right: 0;
 }
 
-.dropdownToggle {
+.toggle {
   display: block;
   padding: 3px;
 }
 
+.menu {
+  max-height: calc(var(--rvr-space-lg) * 2 * 5.5); /* --rvr-space-lg * 2 is the height of one row. */
```

Comment:
might we bring more clarity to this line by putting `2` and `5.5` into some well-named variables? (might not need this comment at that point)

Reply:
Just local variables for clarity? 💯 
I'll be more nervous if you mean global variables. This isn't a universal menu item height, it's reverse-engineered from this particular dropdown usage.

Reply:
just local, ya

Reply:
Done